front_end/ndb/NdbMain.js

```diff
@@ -385,11 +380,18 @@ SDK.TextSourceMap.load = async function(sourceMapURL, compiledURL) {
   const {payload, error} = await Ndb.backend.loadSourceMap(sourceMapURL, compiledURL);
   if (error || !payload)
     return null;
+
+  let textSourceMap;
   try {
-    return new SDK.TextSourceMap(compiledURL, sourceMapURL, payload);
+    textSourceMap = new SDK.TextSourceMap(compiledURL, sourceMapURL, payload);
   } catch (e) {
     console.error(e);
     Common.console.warn('DevTools failed to parse SourceMap: ' + sourceMapURL);
     return null;
   }
+
+  if (textSourceMap._baseURL.startsWith('file://'))
+    for (const mapping of textSourceMap.mappings()) mapping.columnNumber += 62;
```

Comment:
ideas on how to something like this? https://github.com/nodejs/node/issues/17396#issuecomment-348285454. NdbMain doesn't run in node, so can't just do it here.

should maybe do something more clever like this: https://cs.chromium.org/chromium/src/third_party/blink/renderer/devtools/front_end/persistence/Persistence.js?type=cs&q=%22._NodePrefix%22&sq=package:chromium&g=0&l=253 .